front/src/modules/pathfinding/components/Itinerary/ModalSuggeredVias.tsx

```diff
@@ -117,7 +117,7 @@ export default function ModalSugerredVias({
   return (
     <div className="manage-vias-modal">
       <ModalHeaderSNCF withCloseButton>
-        <h1>{`${t('manageVias')} ${vias.length > 0 ? `(${vias.length})` : ''}`}</h1>
+        <h1>{`${t('manageVias', { count: vias.length })}`}</h1>
```

Comment:
You can even remove the extra string ;)
```suggestion
        <h1>{t('manageVias', { count: vias.length })}</h1>
```

Reply:
Good point!